code/modules/research/designs/weapons.dm

```diff
@@ -148,7 +147,6 @@
 	reliability_base = 100
 	category = "Weapons"
 	build_path = /obj/item/weapon/reagent_containers/spray/chemsprayer
-	req_lock_access = list(access_medical, access_cmo)
```

Comment:
Just don't touch code that's not relevant to your PR, regardless of what you're doing with it.